src/common/sorted-table/index.js

```diff
@@ -134,9 +143,40 @@ class ColumnHead extends Component {
 
 // ===================================================================
 
+class Checkbox extends React.PureComponent {
```

Comment:
This class could have extended from `BaseComponent`, but the idea to extend directly from `React.Component` or `React.PureComponent` when the advanced features of `BaseComponent` are not necessary is to make the component reusable outside our project.